### PR TITLE
CCP-001: Add UI and app logic

### PR DESCRIPTION
**Pull request by chornaya-com**
### Description

Add UI and logic to the project

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                    <a href="#" class="clear-btn btn purple lighten-2">Clear All</a>
+                </li>
+            </ul>
+        </div>
+    </div>
+</nav>
+
+<br>
+
+
```
Comment (zaqqaz) [inline src/index.html]: minor: extra line 😉

**Review by zaqqaz — APPROVED** [review]
LGTM 👍
just one minor comment 😉

File: src/index.html (hunk 1 above)
Comment (chornaya-com) [reply src/index.html]: done ✅

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                    <div class="input-field col s6">
+                        <input type="text" placeholder="Add item" id="item-name">
+                        <label for="item-name">Meal</label>
+                    </div>
+                    <div class="input-field col s6">
+                        <input type="text" placeholder="Add calories" id="item-calories">
+                        <label for="item-calories">Calories</label>
+                    </div>
+                    <button class="add-btn btn purple"><i class="fa fa-plus"></i> Add Meal</button>
+                    <!--                    <button class="update-btn btn orange"> <i class="fa fa-pencil-square-o"></i> Update Meal</button>-->
```
Comment (chornaya-com) [inline src/index.html:49]: buttons logic will be implemented later

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                    <!--                    <button class="delete-btn btn red"> <i class="fa fa-remove"></i> Delete Meal</button>-->
```
Comment (chornaya-com) [inline src/index.html:50]: buttons logic will be implemented later

**Pull request merged**